polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaSource.scala

```diff
@@ -265,7 +265,7 @@ class ScalaSource[Interpreter <: ScalaInterpreter](val interpreter: Interpreter)
     } yield context
 
     def completionResults(tree: Tree): Either[Throwable, (global.Type, List[global.Symbol])] = tree match {
-      case global.Select(qual: global.Tree, _) =>
+      case global.Select(qual: global.Tree, _) if qual != null =>
```

Comment:
Is this where we sometimes got those noisy NPEs in completions?

Reply:
Yeah.